walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java

```diff
@@ -268,7 +270,155 @@ public void notEnoughRoom() {
     Collection<TimeRange> actual = query.query(events, request);
     Collection<TimeRange> expected = Arrays.asList();
 
+    Assert.assertEquals(expected, actual);
+  } 
+
+  @Test
+  public void notEnoughRoomForOptional() {
```

Comment:
I would throw the word attendees at the end of this test name, just to be explicit about who or what is optional.

Reply:
Sounds good I threw it on the end there.